### PR TITLE
Provider signup, update photo uploader and employee modals

### PR DESCRIPTION
**Pull request by aghimpu**
https://github.com/teamforus/forus/issues/1464

### CODE REVIEW

**Comment by maxvisser** [comment]
Discussed this with Gerben; He does not like this being merged (changing all the modals style even outside sign-up-v2)

**Comment by maxvisser** [comment]
Needs to be discussed on how to proceed @jamalv

**Comment by jamalv** [comment]
how big is the impact? on how many places will this look off?

on the dashboards I think this xD : https://github.com/teamforus/forus/issues/560

**Comment by maxvisser** [comment]
Modals on the dashboard: generic ones and where we show the cropper. (old cropper is gone)

**Comment by maxvisser** [comment]
Valentin pointed out its only the cropper

**Comment by danrminds** [comment]
Old
Screenshot 2020-05-06 at 22 02 00

New
Screenshot 2020-05-06 at 22 00 38

This is the old vs new version. 
When designing the uploader on signup flow we asked to make small changes on styles only, but the final version on signup flow ended up to need separate HTML as well. 
We (Valentin and me) like the old version better. 
But we also consider - that is necessary to have only 1 version everywhere, to avoid duplicates and dependencies where unnecessary. 

At some point we would have to unify the styles of signup flows and dashboard in general (buttons, fields, forms, popups styles), to not have to maintain lots of sets of components.

**Comment by jamalv** [comment]
TLDR: I agree the old one was better, and I think it needs another iteration.

Full braindump: The second one in the screenshot looks messy. And the upload image call to action is not really clear to me. I do really like the bigger preview in the second one, as this is the core function. I think making an extra iteration on this might be a good move.

**Side note:** We might need a better approval proces on designs. As I think a pr is a bit late in the pipeline to have a discussion about it. I would rather start development later but make sure there is agreement that the new design is an improvement, as now Gerben, Dan, Valentin and Me say it is not, but the code is already here. Could we have also had this discussion based on a Figma prototype while it was still easier to give feedback?

This might be getting off topic, but I’m just thinking it out here:

- I / someone makes an improvement proposal with a rough prototype, and request approval by max.
- when max approves the proposal (basic concept) it is turned into an epic; the designs are finalized into a prototype and tasks are created; approval is requested to me; when I approve; the epic is handed of to teamleaders for development. this way there are more review moments to catch this earlier and save discussion later. 

Long term strategy should be to harmonize al the styles and use the same or similar components where possible. This will give the user a more consistent experience and makes maintaining the codebase easier. When the library in Figma of the current scope is fully finished, we can make a duplicate, and do some experimentation on aligning everything. When we have agreement about the direction we can implement it.

**Comment by maxvisser** [comment]
Allright lets close this pr and try better next time. 

We can talk about this abit more during next retrospective. (or you can make a seperate issue/discussion)
